web/viewProjects.php

```diff
@@ -85,6 +87,28 @@ function areas(val){
 
     };
 
+    var customersStore = new Ext.data.ArrayStore({
+        id: 0,
+        fields: ['id', 'name'],
+        sortInfo: { field: 'name', direction: 'ASC' },
+        data : [
+            <?php
+
+            foreach((array)$customers as $customer)
+                echo "[{$customer->getId()}, '{$customer->getName()}'],";
```

Comment:
There will be problems if the customer name contains a `'` character. You will have to escape those.